docs/templates/parameters.md

```diff
@@ -222,8 +263,11 @@ In some cases, deleting a workspace with a hard dependency on a legacy parameter
 
 > ⚠️ Migration is available until v0.24.0 (Jun 2023) release.
```

Comment:
<i class='graphite__hidden'>[Re: lines 255 to 264]<br/></i>

@bpmct and @mtojek should we remove these institutions altogether?<span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/coder/coder/9139?utm_source=unchanged-line-comment">Graphite</a>.</span>

Reply:
Yes, sure. Migration logic is already gone.